src/Model/EloquentObserver.php

```diff
@@ -87,6 +88,10 @@ public function updatedMultiple(EloquentModel $model)
      */
     public function deleting(EloquentModel $entry)
     {
+        if (!$this->dispatch(new RestrictDelete($entry))) {
+            return false;
```

Comment:
I guess just return would be enough. It is not JS ))

Reply:
You need to `return false` to cancel the detele ;)

Reply:
Hm... No! You don't need to return anything from an observer method.

Reply:
```
        if ($this->fireModelEvent('saving') === false) {
            return false;
        }
```

It is :)

Reply:
It is not! ))) I bet 200$

Reply:
Just tested, my brand was deleted, poor little brand. I'll send you my Paypal :D

Reply:
```
      if ($this->fireModelEvent('saving') === false) {
            return false;
        }
```

Forgot to told you: It is part of the EloquentModel, it doesn't come from my code. It is what cancel the saving() and it is checking for `=== false`.

Reply:
So if the Observer doesn't return false, it will never return anything and the delete will continue. ;)

Reply:
> and it is checking for === false

Show me this checking in the sources.

Reply:
Lazy boy! :D

Reply:
Lazy boy! :D
https://github.com/anomalylabs/streams-platform/blob/8fc6c0ff9df75d2505e9038468edd65217503985/src/Model/EloquentModel.php#L427

Reply:
It only works for `saving`. You have another method.